DuckDuckGo/FeatureFlagging/Model/FeatureFlag.swift

```diff
@@ -66,12 +68,32 @@ extension FeatureFlag: FeatureFlagSourceProviding {
             return .remoteReleasable(.subfeature(AutofillSubfeature.credentialsImportPromotionForExistingUsers))
         case .networkProtectionUserTips:
             return .remoteDevelopment(.subfeature(NetworkProtectionSubfeature.userTips))
+        case .htmlNewTabPage:
+            return .disabled
         }
     }
 }
 
+protocol LocalFeatureFlagOverriding {
+    func localOverride(for featureFlag: FeatureFlag) -> Bool?
+}
+
+extension FeatureFlag: LocalFeatureFlagOverriding {
+    func localOverride(for featureFlag: FeatureFlag) -> Bool? {
+        NSApp.delegateTyped.featureFlagOverrides.override(for: featureFlag)
+    }
+
+    var localOverride: Bool? {
+        localOverride(for: self)
+    }
+}
+
 extension FeatureFlagger {
-    public func isFeatureOn(_ featureFlag: FeatureFlag) -> Bool {
-        isFeatureOn(forProvider: featureFlag)
+
+    public func isFeatureOn(_ featureFlag: FeatureFlag, allowOverride: Bool = true) -> Bool {
+        if allowOverride, let localOverride = featureFlag.localOverride {
+            return localOverride
+        }
+        return isFeatureOn(forProvider: featureFlag)
```

Comment:
my original idea was more on the lines of using isFeatureOn(forProvider: featureFlag) that already checks for internal user with the addition of localOverride (just to avoid having to deal separately with checking it’s an internal user

Reply:
This is perfectly doable, but would require a BSK change (`FeatureFlagger` protocol doesn't define `internalUserDecider`). I focused on showcasing the solution here, so the final implementation is subject to change, but I will make sure to update this.